testutil/rapidproto/rapidproto.go

```diff
@@ -0,0 +1,221 @@
+package rapidproto
+
+import (
+	"fmt"
+
+	"google.golang.org/protobuf/proto"
+	"google.golang.org/protobuf/reflect/protoreflect"
+	"google.golang.org/protobuf/reflect/protoregistry"
+	"gotest.tools/v3/assert"
+	"pgregory.net/rapid"
+)
+
+func MessageGenerator[T proto.Message](x T, options GeneratorOptions) *rapid.Generator[T] {
```

Comment:
does this need to be a generic?

Reply:
I think it's good because rapid uses generics